gemini/use-cases/education/resume_generator.ipynb

```diff
@@ -0,0 +1,877 @@
+        "at the top.\n",
+        "- Keep only the Skills that are relevant to the job description.\n",
+        "- Arrange the Certifications such that the certification that are most relevant to the job\n",
+        "description are at the top.\n",
+        "- Keep only the Certifications that are relevant to the job description.\n",
+        "\"\"\"\n",
+        "\n",
+        "PROMPT_RESUME = \"\"\"\n",
+        "You are a seasoned career advising professional expert in crafting resumes and cover letters.\n",
+        "\n",
+        "Consider the below details that have been extracted from a resume\n",
+        "%s\n",
+        "\n",
+        "Use the above details and craft a resume to satisfy the job description provided below\n",
+        "%s\n",
+        "\n",
+        "Instructions:\n",
+        "- Maintain truthfulness and objectivity\n",
+        "- Proofread and correct spelling and grammatical errors\n",
+        "- Prefer active voice over passive voice\n",
+        "- The output resume should have the following valid JSON format\n",
+        "{\n",
+        "\"Personal_Information\": {\"Name\": \"\", \"Address\": \"\", \"Mobile_No\": \"\", \"Email\": \"\"},\n",
+        "\"Professional_Summary\": \"\",\n",
+        "\"Education\": \"\",\n",
+        "\"Coursework\": \"\",\n",
+        "\"Skills\": \"\",\n",
+        "\"Experience\": \"\",\n",
+        "\"Certifications\": \"\",\n",
+        "\"Languages\": \"\",\n",
+        "\"Extracurricular_Activities\": \"\",\n",
+        "\"References\": \"\",\n",
+        "}\n",
+        "- Generate a concise Professional_Summary based on the job description\n",
+        "- Output Education, Coursework, Skills, Experience, Certifications, Languages,\n",
+        "Extracurricular_Activities and References as numbered lists.\n",
+        "- Arrange the Coursework such that the coursework that are most relevant to the job description are\n",
+        "at the top.\n",
+        "- Arrange the Skills such that the skill that are most relevant to the job description are\n",
+        "at the top\n",
+        "- Arrange the Certifications such that the certification that are most relevant to the job\n",
+        "description are at the top\n",
+        "\"\"\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "a176f957-bbf2-4d4c-9ab6-82d6f691adfa"
+      },
+      "outputs": [],
+      "source": [
+        "# Resume structure\n",
+        "resume_structure = [\n",
+        "    \"Personal_Information\",\n",
+        "    \"Professional_Summary\",\n",
+        "    \"Education\",\n",
+        "    \"Coursework\",\n",
+        "    \"Skills\",\n",
+        "    \"Experience\",\n",
+        "    \"Certifications\",\n",
+        "    \"Languages\",\n",
+        "    \"Extracurricular_Activities\",\n",
+        "    \"References\",\n",
+        "]"
```

Comment:
See if you can use this list as a variable input to the prompt to define the structure instead of copy/pasting.

Reply:
Used this list's items in prompt